### PR TITLE
Fix/interface

### PR DESCRIPTION
**Pull request by koreanddinghwan**

### CODE REVIEW

File: src/modules/notification/templates/baseMail.template.ts
Comment (koreanddinghwan) [inline src/modules/notification/templates/baseMail.template.ts:1]: #126

File: src/models/user/queries/userSelect.query.ts
Comment (koreanddinghwan) [inline src/models/user/queries/userSelect.query.ts:1]: #127

File: src/models/user/dto/response/userGetResponse.dto.ts
Comment (koreanddinghwan) [inline src/models/user/dto/response/userGetResponse.dto.ts:1]: #127

File: src/common/interfaces/api/mentorProfile/mentorProfileRequest.interface.ts
Comment (koreanddinghwan) [inline src/common/interfaces/api/mentorProfile/mentorProfileRequest.interface.ts:1]: fix typo

**Comment by koreanddinghwan** [comment]
this will resolves #126 , #127

**Review by JuneParkCode — APPROVED** [review]

**Pull request merged**